src/client/renderingengine.h

```diff
@@ -29,6 +29,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // include the shadow mapper classes too
 #include "client/shadows/dynamicshadowsrender.h"
 
+struct VideoDriverInfo {
+	std::string name;
```

Comment:
cannot we use a (const) char * directly here ? as we recast it after, if not, i'm fine it's clearly better

Reply:
`std::string` is a wrapped char* anyway, but using an actual string type declares the intent better. Is that fine?

Reply:
a `std::string` necessarily allocates memory on the heap doesn't it?